### ISSUE TITLE
Anit map.cfg Modifying

### ISSUE THREAD

**Issue by SF-Fatality** [inexor-game/code#167]
Currently in Sauer, you can easily modify maps through their respective .cfg file without the map appearing as a "modified map" on servers, as long as you don't edit the .ogz itself. This is the easiest and most common "cheat" in Sauer as far as I'm aware, because you can simply put "mapmodelreset" in the bottom of "dust2.cfg", and magically you have no boxes on dust2, without a modified map. You can also change things like fog values, water values (ex. making it possible to see people on tortuga much easier), change the skybox on any map, and even change the lighting (but with this you would have to do a /calclight everytime the map loads, which IS possible in  non-edit modes). It would be nice to have some way where the game or server or whatever would recognize if the player has a modded .cfg or not, just like with modified .ogz's. As a mapper, I do not like the thought of someone changing my fog, for example, on a map where it is supposed to be very present :P

**Comment by Croydon** [comment]
Depends on #53. Imo doesn't make any sense to implement this before we have the new map format. Furthermode, since the new map format will require just one file for a map, such a check, if the map is modifying, will be lot easier anyway.

**Comment by SF-Fatality** [comment]
Cool

**Comment by chasester** [comment]
Technically all the user has to do is send the correct map to the server, then have the game run the modded map. I would suggest server side checks for things like player position bullet movement etc. so that will limit the usefullness of the cheating.
If you dont want any models all you have to do is delete the model data (obj md5 etc) this will remove any clipping on them :+1:
